src/Alert.php

```diff
@@ -27,23 +27,43 @@
  */
 final class Alert extends Widget
 {
+    public const TYPE_PRIMARY = 'alert-primary';
```

Comment:
Why do we need constants to be public, if they have individual methods that set their value, you can even remove the constants and assign the values directly to each method.